ci/helm/values/values-kafka.yaml.gotmpl

```diff
@@ -0,0 +1,67 @@
+image:
+  tag: "3.6.0"
+kraft:
+  enabled: false
+controller:
+  # required to allow Zookeeper mode
+  replicaCount: 0
+listeners:
+  client:
+    protocol: PLAINTEXT
+  interbroker:
+    protocol: PLAINTEXT
+broker:
+  replicaCount: 1
+  persistence:
+    enabled: false
+  initContainers:
+  - name: init-zookeeper
+    image: busybox
+    imagePullPolicy: IfNotPresent
+    command: ['sh', '-c', 'until nc -w1 {{"{{"}} printf "%s%s" (include "kafka.zookeeper.fullname" .) (tpl .Values.zookeeperChrootPath .) | quote {{"}}"}} {{"{{"}} .Values.zookeeper.service.ports.client {{"}}"}}; do echo "waiting for zookeeper"; sleep 2; done;']
+  resources:
+    requests:
+      cpu: "1"
+      memory: "1Gi"
+    limits:
+      memory: "1Gi"
+  extraConfig: |
+    auto.create.topics.enable=false
+    delete.topic.enable=true
+    group.initial.rebalance.delay.ms=100
+    log.retention.minutes=24
```

Comment:
here also configuration for unit test, but might be to short for a benchmark

Reply:
This will be removed, but this makes me think that I copied/pasted this block from nuxeo-lts, and it seems it applies for nuxeo-lts unit tests but also for the benchmark, it may be worth checking it 🤔.